crates/plugins/src/store.rs

```diff
@@ -24,6 +24,15 @@ impl PluginStore {
     }
 
     pub fn try_default() -> Result<Self> {
+        if let Ok(brew_prefix) = std::env::var("HOMEBREW_PREFIX") {
+            let plugins_dir = Path::new(&brew_prefix).join("var").join("spin").join("plugins");
+
+            if plugins_dir.exists() && plugins_dir.is_dir() {
```

Comment:
`is_dir` implies `exists`
```suggestion
            if plugins_dir.is_dir() {
```